### PR TITLE
Blog: The Death of Point-to-Point: Why You Need a Unified Namespace

### LINKED ISSUE #2797: Blog Art : UNS: Why Point-to-point connection is dead
### Related Issue

https://github.com/FlowFuse/website/pull/2794

### Description

**Considerations:**
- Blog Tile
- A separate graphic illustrating the point-to-point spaghetti, similar to the following image.

**Inspirations:**

Screenshot_2024-11-21_at_4 13 27_PM-removebg-preview

### PR DESCRIPTION
**Pull request by sumitshinde-84**
## Related Issue(s)

closes #2797

## Checklist

 - [ ] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [ ] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

File: src/blog/2024/11/why-point-to-point-connection-is-dead.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
+---
+title: "UNS: Why Point-to-point Connection is Dead"
```
Comment (Hasmin-AC) [inline src/blog/2024/11/why-point-to-point-connection-is-dead.md:2]:
```suggestion
title: The Death of Point-to-Point: Why You Need a Unified Namespace"
```

File: src/blog/2024/11/why-point-to-point-connection-is-dead.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+subtitle: Why the Traditional Point-to-Point Connection Model is Holding Back Manufacturing Innovation and How Unified Namespace (UNS) Can Transform Factory Operations
```
Comment (Hasmin-AC) [inline src/blog/2024/11/why-point-to-point-connection-is-dead.md:3]:
```suggestion
subtitle: Traditional Point-to-Point Connection Model is Holding Back Manufacturing Innovation - Unified Namespace (UNS) Can Transform Factory Operations
```

File: src/blog/2024/11/why-point-to-point-connection-is-dead.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+authors: ["sumit-shinde"]
+image: /blog/2024/11/images/why-point-to-point-connection-dead.png
+keywords: uns, cons of p2p, cons of point-to-point connection, drawbacks of point-to-point connection, publish-subscribe, mqtt
+tags:
+   - flowfuse
+   - mqtt 
+   - unified namespace
+---
+
+Manufacturing has changed significantly over the years, driven by new technology and the need for better communication between systems. In the past, point-to-point (P2P) connections—where devices communicate directly with each other—were the standard. However, as factories become more complex, P2P connections are no longer practical. This article explains why P2P connections are outdated and how a Unified Namespace (UNS) offers a better, more flexible solution.
```
Comment (Hasmin-AC) [inline src/blog/2024/11/why-point-to-point-connection-is-dead.md:15]:
```suggestion
Manufacturing has changed significantly over the years, driven by new technology and the need for better communication between systems. In the past, point-to-point (P2P) connections, where devices communicate directly with each other, were the standard. However, as factories become more complex, P2P connections are no longer practical. This article explains why P2P connections are outdated and how a Unified Namespace (UNS) offers a better, more flexible solution.
```

**Review by Hasmin-AC — APPROVED** [review]

File: src/blog/2024/11/why-point-to-point-connection-is-dead.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+
+When scaling a P2P network, changes made to one part of the system often trigger a ripple effect throughout the entire network. For instance, adding a new sensor may require updating the controller, the SCADA system, and even the Manufacturing Execution System (MES). This cascading need for updates across different layers of the network makes scaling more complicated and costly. Furthermore, the introduction of new devices means additional configuration and troubleshooting, often leading to disruptions in operations and extended downtimes while the new devices are integrated.
+
+These issues compound when scaling across multiple production lines or sites, creating an increasingly complex web of p2p connections. As the number of devices grows, so does the risk of errors, network failures, and delays. This makes it difficult for manufacturers to respond to the growing demands of production while maintaining efficiency, reliability, and uptime.
+
+## Unified Namespace: The Modern Solution
+
+![Hub and Spoke Model](./images/hub.png){data-zoomable}
```
Comment (Hasmin-AC) [inline src/blog/2024/11/why-point-to-point-connection-is-dead.md:76]: This image appears rather large. I would consider reducing the size.

**Comment by sumitshinde-84** [comment]
@Yndira-E, could you please update the title in the tile? It has been changed in the review.

File: src/blog/2024/11/why-point-to-point-connection-is-dead.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+
+As manufacturing moves toward more modern, interconnected approaches like Industry 4.0, the limitations of point-to-point connections become more apparent. Here's why P2P connections are no longer sufficient for today’s manufacturing environments:
+
+### 1. Limited Data Sharing, Visibility, and Delayed Data
+
+Point-to-point connections often lead to data silos, where devices or machines communicate only with their immediate hierarchy level. This isolation severely limits system-wide visibility, making it challenging to share critical data in real-time. Consequently, problems such as defective products, unexpected downtime, and the need for rework may go unnoticed until the damage is done.
+
+In modern manufacturing, real-time data is indispensable for maintaining operational efficiency. However, point-to-point connections introduce significant delays, as data must traverse multiple layers—control systems (Level 1), supervisory control (Level 2), manufacturing execution systems (Level 3), and finally, higher-level management systems (Level 4) as outlined in the ISA-95 model. Each additional layer compounds latency, slowing response times and postponing the detection of issues.
```
Comment (ZJvandeWeg) [inline src/blog/2024/11/why-point-to-point-connection-is-dead.md:38]:
```suggestion
In modern manufacturing, real-time data is indispensable for maintaining operational efficiency. However, point-to-point connections introduce significant delays, as data must traverse multiple layers—control systems (Level 1), supervisory control (Level 2), manufacturing execution systems (Level 3), and finally, higher-level management systems (Level 4) as outlined in the ISA-95 model. Each additional layer compounds latency, slowing response times and postponing the detection of issues, see also our post on [Why the Automation Pyramid slows you down](https://flowfuse.com/blog/2023/08/isa-95-automation-pyramid-to-unified-namespace/).
```

**Comment by Yndira-E** [comment]
@sumitshinde-84 updated tile in this PR: https://github.com/FlowFuse/website/pull/2812

**Pull request merged**